hack/update-versions.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -eu -o pipefail
+
+dir=$1
+version=$2
+
+find "$dir" -type f -name '*.yaml' | while read -r f ; do
+  sed "s|version: latest|version: ${version}|" "$f" > .tmp
```

Comment:
what happens when the version has been committed and is no longer latest and then changes?

Reply:
Are you asking to make this script idempotent? It currently is not as you so indicate.

I use envsubst in all my projects which makes it easy to do something like this- but I know a lot of people like being able to apply the base manifest as is.

I'll figure something out to satisfy being idempotent while keeping with the argo style.  I think I had explicitly matched on latest as I didn't want to match on `version: v1alpha1`